gitlab/mixins.py

```diff
@@ -99,8 +99,7 @@ def get(
             GitlabAuthenticationError: If authentication is not correct
             GitlabGetError: If the server cannot perform the request
         """
-        if not isinstance(id, int):
-            id = utils._url_encode(id)
+        id = utils._url_encode(id)
         path = f"{self.path}/{id}"
```

Comment:
Just from a quick glance it would seem to me we could use the property here as well, or did I probably miss something?

```suggestion
        path = f"{self.path}/{self.encoded_id}"
```

Reply:
No, because we need to store the encoded value below when doing lazy.

Reply:
Oh and this is `id` not `self.id`

Reply:
My bad, I'll look closer next time :laughing: